### PR TITLE
[Case Study] Add and update case studies

### PR DESCRIPTION
**Pull request by Sherlock113**
Add some case studies to the case study page.

I also updated the BIGO and Keytop case studies by replacing the Chinese videos with English articles for a global audience.

Previews:
Screen Shot 2023-01-09 at 11 19 09
Screen Shot 2023-01-09 at 11 18 56
Screen Shot 2023-01-09 at 11 18 44
Screen Shot 2023-01-09 at 11 18 32
Screen Shot 2023-01-09 at 11 18 16

### CODE REVIEW

**Comment by Sherlock113** [comment]
@tisonkun @merlimat PTAL. Thanks.

File: src/pages/case-studies.js [hunk 1]
```diff
@@ -214,6 +226,12 @@ const csObj =
         image: require("@site/static/img/getui.png").default,
         description: "We adopted Pulsar for the new priority-based push notification solution."
       },
+      {
+        company: "NTT Software Innovation Center",
+        link: 'https://www.youtube.com/watch?v=o6Dt_LZCd3U',
+        image: require("@site/static/img/ntt_logo.jpg").default,
+        description: "Pulsar has a flexible design and its performance is already good enough for many use cases. The NTT Software Innovation Center ran different performance tests and implemented their own subscription model in Pulsar to further improve its performance. This has allowed them to use one Pulsar topic to support 100K consumers in their IoT scenario."
```
Comment (Anonymitaet) [inline src/pages/case-studies.js:233]:
```suggestion
        description: "Pulsar has a flexible design and its performance is already good enough for many use cases. The NTT Software Innovation Center ran different performance tests and implemented its own subscription model in Pulsar to further improve its performance. This has allowed them to use one Pulsar topic to support 100K consumers in their IoT scenario."
```

File: src/pages/case-studies.js [hunk 2]
```diff
@@ -250,9 +268,9 @@ const csObj =
     transportation_logistics: [
       {
         company: "Keytop",
-        link: 'https://www.youtube.com/watch?v=ITR-J_D8wFg',
+        link: 'https://streamnative.io/success-story/keytop/',
         image: require("@site/static/img/keytop.png").default,
-        description: "Pulsar is an ideal streaming data platform for our parking system. We customize a messaging system with EMQX, Pulsar and Sink to deal with our data."
+        description: "Keytop redesigned its message system by using Apache Pulsar as the backbone of its architecture. Its geo-replication feature helps Keytop securely store mission-critical data across multiple data centers."
```
[the remark this answers is not in the document]
Comment (Sherlock113) [reply src/pages/case-studies.js]: Thanks. Updated.

File: src/pages/case-studies.js (hunk 1 above)
Comment (Sherlock113) [reply src/pages/case-studies.js]: Thanks. Updated.

**Review by tisonkun — APPROVED** [review]
Cool!

**Review by Anonymitaet — APPROVED** [review]

**Pull request merged**